i18n/en.json

```diff
@@ -6,5 +6,6 @@
 		]
 	},
 	"scryfalllinks-desc": "Creates Scryfall links from Magic: The Gathering card names.",
-	"scryfalllinks-unrecognized-card": "Unrecognized card"
+	"scryfalllinks-unrecognized-card": "Unrecognized card",
+	"scryfalllinks-card-tooltip-error": "Error, unable to obtain card details"
```

Comment:
Can you trim this error message back down to just one or two words? I think this isn't conveying much more than "Error" does.